openapi/components/schemas/ThreeDSecureServers/EMerchantPay3dsServers/eMerchantPay3dsServers.yaml

```
@@ -1,16 +1,12 @@
 description: eMerchantPay 3DS servers.
```

question: should this be singular or plural? 
```suggestion
description: eMerchantPay 3DS server.
```